### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Posandu**

### CODE REVIEW

**Review by Posandu — COMMENTED** [review]
Done

File: README.md [hunk 1]
```diff
@@ -1,48 +1,55 @@
-You will need to supply your own Discord Bot Token (found at https://discord.com/developers/), Client ID (found once you create a Discord Application), and Guild ID (found when you enable Developer Mode in Discord Settings in the Main Client, then copy your server, which is also known as a guild, ID). Other than that, this Dockerfile is usable OOTB (out of the box).
+You need to supply your own
+1. Discord Bot Token (found at https://discord.com/developers/)
+2. Client ID (found once you create a Discord Application)
+3. Guild ID (found when you enable Developer Mode in Discord Settings in the Main Client
```
Comment (Lukas-Batema) [inline README.md:4]: Merge lines 4 and 5 up to the, "...guild, ID)." and remove the comma after guild and the period after the closed parenthetical.

File: README.md [hunk 2]
```diff
@@ -1,48 +1,55 @@
...
 # How to Create a Discord Bot, Add it To Your Server, and Get it Running
-To create the bot you will want to go to https://discord.com/developers/. To create an application, which we want to do, click, `New Application`.
-<br>
-Now once we have it created, we will want to create the bot. We do this by heading over to the `Bot` tab (found on the left side bar of the site) and click `Add Bot`.
-<br>
-Now we will add the permissions we need. head on over to `OAuth2 -> URL Generator`. The below picture will tell you what permissions we will need.
-<br>
+
+To create the bot, you will want to go to https://discord.com/developers/. To create an application, which we want to do, click, `New Application`.
+
```
Comment (Lukas-Batema) [inline README.md:12]: Keep all the `<br>`s.

**Review by Lukas-Batema — CHANGES_REQUESTED** [review]

**Review by Lukas-Batema — APPROVED** [review]
Looks good now

**Pull request merged**